Makefile

```diff
@@ -404,3 +404,11 @@ integration-tests: all testrunner
 #     integration-tests
 #
 	RUNNER_ARGS="-parallel" ./test/run-integration-tests.sh
+
+netlify-site-preview:
+	@mkdir -p site-build
```

Comment:
1. Do we want to delete this folder at the end of the `netlify-site-preview` target?
2. Do we want to `git`-ignore it?